### DISCUSSION TITLE
Low Audio Volume

### DISCUSSION THREAD

**Discussion by Chase-Ehlert** [LizardByte/community#227] [Sunshine] [answered]
I have a Sunshine and Moonlight streaming setup and everything is working fine besides the volume level of the audio coming to the client. I have configured Sunshine to use the "audio sink" that my desktop headset uses but because the system volume for the headset is set low, I have to turn my TV volume up to almost max to hear anything. Is there something obvious that I'm missing? I don't want to have to turn up the volume to max on my PC or TV anytime I'm going to stream.

**Comment by ReenigneArcher** [comment, marked as answer]
Use a different audio device. You can set the device in the config. I'd probably suggest the virtual sink option. https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/advanced_usage.html#audio

**Comment by Chase-Ehlert** [comment]
I have tried the Realtek driver that I have installed (different from what
my default is for my headset) but then the issue is I'll have to switch
what audio output my PC is using whenever I go back to it.

I have also tried using the default virtual sink but I don't think it works
as I can't get a session to work correctly with that config. I couldn't
find any method to confirm that the virtual sink is the correct one for my
computer.
